src/common/framework/query/query.ts

```diff
@@ -207,7 +207,20 @@ Expectation should be of the form path/to/cts.html?worker=0&q=suite:test_path:te
       expectationQuery = parseQuery(entry.query);
     }
 
-    if (compareQueries(query, expectationQuery) === Ordering.Unordered) {
+    // Strip params from multicase expectations so that an expectation of foo=2;*
+    // is stored if the test query is bar=3;*
+    const queryForFilter =
+      expectationQuery instanceof TestQueryMultiCase
```

Comment:
uNit: I prooobably would have done this on the query instead of the expectationQuery but I think they're equivalent